base/task.jl

```diff
@@ -365,6 +365,77 @@ function wait(t::Task)
     nothing
 end
 
+# Wait multiple tasks
+waitany(tasks) = _wait_multiple(tasks)
+waitall(tasks; failfast=false) = _wait_multiple(tasks; all=true, failfast=failfast)
```

Comment:
Can you add docstrings to these, add them to docs/src/parallel.md, and add a mention of this feature in NEWS.md? I am quite happy with where this PR is, so just these few remaining details to cleanup. Thanks for pursuing this!

Reply:
Yes, I intended to add docstring to these after finishing the fixes to the implementation.

I think I can finish the remaining fixes and writing docs in this week.  Thank you for your valuable feedback! Without them, I couldn’t  complete this pull request.